starter-code/views/drones/index.ejs

```diff
@@ -0,0 +1,9 @@
+<h2> Drone City </h2>
```

Comment:
Looks good! Just change the route in the drones.js file to `'/drones'` and it should show up!